src/components/alert/SpshAlert.vue

```diff
@@ -26,26 +27,27 @@
   <v-container v-if="modelValue">
     <v-slide-y-transition>
       <v-alert
+        @click:close="closeAlert"
+        :closable="closable"
+        :data-testid="`${dataTestIdPrefix ? dataTestIdPrefix : 'spsh'}-alert`"
```

Comment:
```suggestion
       :data-testid="`${dataTestIdPrefix || 'spsh'}-alert`"
```
In case you want a more concise check.